src/comptime.rs

```diff
@@ -5,9 +5,17 @@ use const_soft_float::soft_f64::SoftF64 as Sf64;
 
 /// A `const` matrix type, with dimensions checked at compile time
 /// for all operations.
-#[derive(Clone, PartialEq, PartialOrd, Copy)]
+#[derive(Clone, Copy, PartialEq, PartialOrd)]
```

Comment:
I can revert this ordering but in general it makes it a bit more readable if Clone and Copy are right next to each other.